tonic/handler.go

```diff
@@ -0,0 +1,323 @@
+package tonic
+
+import (
```

Comment:
can you rename to tonic.go ?
diff is very large and hard to review, + mimicking the package name makes it the go-to file when browsing the codebase.

Reply:
Won't fix.